src/consensus/raft/raft.h

```diff
@@ -271,6 +275,23 @@ namespace raft
       become_leader();
     }
 
+    void force_become_follower(Index index, Term term)
```

Comment:
```suggestion
    void init_as_follower(Index index, Term term)
```

We have too many things called force_become, which imply that we start out as something else and perform an arbitrary transition. Here, we know we initialise from a snapshot, there's nothing forced about that.